admin_actions/admin.py

```diff
@@ -45,6 +52,7 @@ def action(
     name: str = None,
     show_message=True,
     visible: VisibleCallback = always_visible,
+    html_class: Optional[str] = None,
```

Comment:
it's good to have configuration, but let's add defaults that makes sense in most cases
```suggestion
DEFAULT_HTML_CLASSES = f"django-admin-action {action_name}"

...
    extra_html_classes: Optional[str] = None,
    ...
    
    html_classes = DEFAULT_HTML_CLASSES
    if extra_html_classes:
    	html_classes = f"{html_classes} {extra_html_classes}"
    html_classes = html_classes.format(action_name=f.__name__)
```